ndcube/tests/test_global_coords.py

```diff
@@ -0,0 +1,19 @@
+import astropy.units as u
+
+import ndcube
+from ndcube.global_coords import GlobalCoords
+
+gc = GlobalCoords(ndcube)  # Initialize an empty GlobalCoords instance
```

Comment:
This shouldn't be passed the module, it should be passed an *instance* of the `NDCube` class.

```suggestion
@pytest.fixture
def global_coords(ndcube_3d_ln_lt_l):
    return GlobalCoords(ndcube_3d_ln_lt_l)
```

(remember to add the import)
```